roles/nfs/Readme.md

```diff
@@ -0,0 +1,78 @@
+Role Definition
+-------------------------------
+- Role name: NFS
+- Definition:
+  - Network File System (NFS) is a distributed file system protocol allowing a user on a client computer to access files over a computer network much like local storage is accessed. 
+  - GPFS file systems may be exported using the Network File System (NFS) protocol from one or more nodes. 
+  - After export, normal access to the file system can proceed from GPFS cluster nodes or NFS client nodes.
+  
+  
+ 
+Prerequisite
+----------------------------
+- CES should be enabled and configured.
+- Atleast 1 node should be designated as nfs node.
+- Native nfs service should be stopped, before usinf this role.
+- nfs-kernel-server service should be stopped.
+- knfs-server service should be stopped
+- The packages required for nfs installation are as follows:
```

Comment:
@shubh1410 Please add here ansible playbook will automatically install this package .

Reply:
@rajan-mis suggested changes implemented